src/utils/vscode/getPRsToPaintPerSHAs.ts

```diff
@@ -34,8 +37,22 @@ export default async function getPRsToPaintPerSHAs({
   // takes the first 22 shas and creates a list to send to the gh api
   let joinedArrayOfSHAs = arrayOfSHAs.slice(0, 22).join();
   if (joinedArrayOfSHAs.length < 1) {
-    noLinesSelected();
-    return { errorText: "No lines selected" };
+    // Running an experiment to see how users behave without this error prompt
+    // But rather running a git blame for the whole class
+    // noLinesSelected();
+    // return { errorText: "No lines selected" };
+    let gitAPI = await getGitAPI();
```

Comment:
This await was already added before calling this function, why add more time to the query?

Reply:
Where is `getGitAPI` getting called? I don't see it

Reply:
https://github.com/watermelontools/wm-extension/blob/3b08fdfcfa859cf59bd74426c9b47fcb11056562/src/extension.ts#L40